trs_filer/ga4gh/trs/endpoints/registerObjects.py

```diff
@@ -0,0 +1,73 @@
+"""Controller for adding new objects."""
+
+from random import choice
+from typing import Dict
+from trs_filer.app import logger
+from flask import (current_app, request)
+from pymongo.errors import DuplicateKeyError
+
+
+class CreateToolPostObject:
```

Comment:
Don't forget to add docstrings for your class and methods.

Reply:
Hate to be picky about this, but there should be empty lines between sections in Google-style docstrings.

Here's an example:

```py
def my_func(arg_1: str = "something") -> str:
    """Summary comment.

    Additional comments on usage etc., if applicable.

    Args:
        arg_1: Description of argument.

    Returns:
        Modified string.

    Raises:
        TypeError: User supplied argument of wrong type.
    """
    if isinstance(arg_1, str):
        return 2 * arg_1
    else:
        raise TypeError("Supplied argument is not of type `str`")
```